src/conv-core/convcore.C

```diff
@@ -3798,19 +3809,21 @@ void ConverseCommonInit(char **argv)
   CmiIOInit(argv);
 #endif
   if (CmiMyPe() == 0)
-      CmiPrintf("Converse/Charm++ Commit ID: %s\n", CmiCommitID);
+    CmiPrintf("[%d] Converse/Charm++ Commit ID: %s\n", CmiMyPe(), CmiCommitID);
```

Comment:
Is it necessary to print the PE if we test `if (CmiMyPe() == 0)` first?

Reply:
done in fixup